src/Illuminate/Mail/Mailer.php

```diff
@@ -383,7 +383,7 @@ protected function parseView($view)
      * Send a Swift Message instance.
      *
      * @param  \Swift_Message  $message
-     * @return void
+     * @return int|null
```

Comment:
the mandril driver for example doesn't return, so developer should NOT reply on the return value here, which is why we made it voide